frontend/src/locales/en/experiments.json

```diff
@@ -165,12 +167,11 @@
   "stepCorrespondsExec": "This step corresponds to execution",
   "stepStateMessage1": "This step is in",
   "stepStateMessage2": "state with this message",
-  "its_plural": "their",
-  "taskDetail":"Task Details",
-  "thisexperiment_plural": "these experiments",
+  "taskDetail": "Task Details",
+  "thisexperiment": "this experiment",
   "thisExperimentToIts": "this experiment to its",
   "thisExperimentToIts_plural": "this experiment to their",
-  "thisexperiment": "this experiment",
+  "thisexperiment_plural": "these experiments",
```

Comment:
Please fix the capitalization of the variable name, here and wherever this is used